toolkit/tools/grapher/grapher.go

```diff
@@ -42,6 +42,7 @@ var (
 	usePreviewRepo                = app.Flag("use-preview-repo", "Pull packages from the upstream preview repo").Bool()
 	disableDefaultRepos           = app.Flag("disable-default-repos", "Disable pulling packages from PMC repos").Bool()
 	ignoreVersionToResolveSelfDep = app.Flag("ignore-version-to-resolve-selfdep", "Ignore package version while downloading package from upstream when resolving cycle").Bool()
+	repoSnapshotTime              = app.Flag("repo-snapshot-time", "Repo time limit for tdnf virtual snapshot").String()
```

Comment:
Id say either plumb this through from the makefile, or omit it. This is  `$(graph_file):...` in `pkggen.mk`. Add a dep on `$(depend_REPO_SNAPSHOT_TIME)` to that as well (no need to clean up anything, the file should regenerate cleanly if needed with just the basic dep).

Reply:
added